### DISCUSSION TITLE
Disable Code Example Powered by scalar.com from the Editor rather than OpenAPI Spec

### DISCUSSION THREAD

**Discussion by Will-Carrara** [GitbookIO/community#597] [Feature Requests]
<img width="1792" alt="image" src="https://github.com/GitbookIO/community/assets/43623732/b523d960-0ff1-4eaf-8e14-acc5641e3c44">

Is it possible to have an option in the editor to disable this javascript example that generates? They don't generate correctly and are misleading in terms of what type of request I want the user to make. I would like to request this feature be added to the editor please. Thank you for your consideration. Example can be found [here](https://openet.gitbook.io/docs/reference/api-reference/raster).

**Comment by addisonschultz** [comment]
Hi @Will-Carrara - good suggestion, but I don't know how fast the team can get to this given it's possible already through the API Spec. Is there a main reason this would be more useful in the editing phase vs in the spec?

**Reply by Will-Carrara** [reply]
Hi @addisonschultz, there is a really popular API framework called fastAPI which generates an OpenAPI JSON dynamically. One of the things I really like about gitbook is the ability to feed in a link to that json spec which automatically updates the documentation when I make changes to our API. The problem arises in that unless I feed a static json in (which in the past fails when as it is a large file size) I lose the core functionality. 

Let me know if that makes sense. Thank you again!

**Reply by addisonschultz** [reply]
Just to make sure I understand correctly 😄 

It's not possible to generate the options here: https://docs.gitbook.com/content-editor/blocks/openapi#options

When the spec is dynamically generated by FastAPI - is that right?
